docs/learn/learn-governance.md

```diff
@@ -246,19 +248,12 @@ votes = tokens * conviction_multiplier
 The conviction multiplier increases the vote multiplier by one every time the number of lock periods
 double.
 
-| Lock Periods | Vote Multiplier | Length in Days |
-| :----------: | :-------------: | :------------: |
-|      0       |       0.1       |     {{ vote_locking_period_0 }}  |
-|      1       |        1        |     {{ vote_locking_period_1 }}  |
-|      2       |        2        |     {{ vote_locking_period_2 }}  |
-|      4       |        3        |     {{ vote_locking_period_3 }}  |
-|      8       |        4        |     {{ vote_locking_period_4 }}  |
-|      16      |        5        |     {{ vote_locking_period_5 }}  |
-|      32      |        6        |     {{ vote_locking_period_6 }}  |
-
+<VLTable network="polkadot"/>
```

Comment:
Here is the custom Voluntary Locking Table component.  Required because you can't load a component inside of the markdown table.